jetty-ee10/jetty-ee10-websocket/jetty-ee10-websocket-jetty-common/src/test/java/org/eclipse/jetty/ee10/websocket/common/MessageOutputStreamTest.java

```diff
@@ -34,12 +36,14 @@
     private static final Logger LOG = LoggerFactory.getLogger(MessageOutputStreamTest.class);
     private static final int OUTPUT_BUFFER_SIZE = 4096;
 
-    public TestableLeakTrackingBufferPool bufferPool = new TestableLeakTrackingBufferPool("Test");
+    // TODO: restore leak tracking.
+    public RetainableByteBufferPool bufferPool = new ArrayRetainableByteBufferPool();
 
     @AfterEach
     public void afterEach()
     {
-        bufferPool.assertNoLeaks();
+        // TODO: restore leak tracking.
+//        bufferPool.assertNoLeaks();
```

Comment:
Why have these tests been commented out but you have a different change for the same ee9 test?

Reply:
Good catch, restore here too.